.bashrc

```diff
@@ -0,0 +1,4 @@
+bundle install --path vendor/bundle
```

Comment:
You don't need to commit the `.bashrc` file, can you please omit it from the PR.

Reply:
Okay, we have resolved it. The bash file has been removed.